modules/virt-what-you-can-do-with-virt.adoc

```diff
@@ -6,8 +6,7 @@
 [id="virt-what-you-can-do-with-virt_{context}"]
 = What you can do with {VirtProductName}
 
-{VirtProductName} is an add-on to {product-title} that allows you to run and manage
- virtual machine workloads alongside container workloads.
+{VirtProductName} is a feature of the {product-title} that you can use to run and manage virtual machine workloads alongside container workloads.
```

Comment:
No need to have the 'the' before {product-title}

Reply:
Fixed.